packages/rocketchat-lib/server/lib/debug.js

```diff
@@ -56,7 +56,6 @@ const wrapMethods = function(name, originalHandler, methodsMap) {
 		const args = name === 'ufsWrite' ? Array.prototype.slice.call(originalArgs, 1) : originalArgs;
 		logger.method(name, '-> userId:', Meteor.userId(), ', arguments: ', args);
 
-		this.unblock();
```

Comment:
Please do not remote this, add a condition to bypass the reset password method.

Reply:
resetPassword is not the only method affected by this:

https://github.com/RocketChat/Rocket.Chat/issues/12762

Reply:
@Hudell do we know if it was generated by a meteor update or by converting the packages?

Reply:
I checked the relevant files on the meteor github and there didn't seem to be any recent change, but I'll look on their history to make sure.

Reply:
As I said, the meteor methods block by default, should be a good reason to think about, its a false impression of "performance", if we have a method that takes much time we should unblock or fix them in special...

Reply:
> Please do not _remote_ this, add a condition to bypass the reset password method.

it a generic method testing specific case, how much time until we get a big switch case and a lot of specific cases?

Reply:
looks like there many methods breaking because of `this.unblock()` (at least `resetPasswordWithTOTP`, `verifyEmail`, `resetPassword` that were mentioned on related issues), so it might safer to remove it by now since we don't know how many other might be broken.

Reply:
but since we *need* to release a fix for those issues, I'd say we can add a bypass to those methods, cut a release and start planing to remove this `unblock()` for the next release. wdyt?

Reply:
https://github.com/meteor/meteor/blob/master/packages/ddp-common/method_invocation.js#L89 looks like this error has existed for over a year.  File hasn't been changed.  So we must have been getting lucky some how?

Reply:
We need to pick some solution, since effecting some of the important operations users do sometimes daily 😬